scrapy/czlscrape/__init__.py

```diff
@@ -1,7 +1,11 @@
 import os
+import time
+import logging
 
 if 'SENTRY_DSN' in os.environ:
     import logging
     from raven.handlers.logging import SentryHandler
     from raven.conf import setup_logging
     setup_logging(SentryHandler(os.environ['SENTRY_DSN'], level=logging.WARN))
+
+logging.Formatter.converter = time.gmtime
```

Comment:
> what about datetime.utcnow() ? That should give us the right value.

Argh, sorry guys, I didn't notice the final `Z` was not `%Z`. In fact both `time.gmtime` and `time.localtime` set timezone information. (though the first one is "GMT", and "Z" is prettier).

> Regarding timezones, I understand what you're saying about convenience of reading the time and date in our own timezone, but this is more of a presentation concern rather than what it should be stored in.

Yes, entirely presentational, but I'd much rather read it in my local time when digging through debug logs chasing for some bug. Not a serious concern though, just me being constructively lazy.

> Your suggestion to use the same timezone on all machines is good, but it should be applied with UTC as timezone. Hardware clocks should be set to UTC, because it is unambiguous.

Of course the hardware clock should be UTC, unless it's, ahem, windows.. What I'm saying is the local timezone might be set to ours, unless someone is particularly interested in machine events relative to the eu-west time. (that means the devops, and this should ultimately be their decision)

> Also, storing datetimes in anything other than UTC means that we need to store both the value and the timezone offset.

I think we should do that anyway. Saves us from ambiguity in the future.

> You are right, we will need to parse the dates in the scraped data and interpret it as being in local Romanian time.

About that, I updated my comment. We don't deal with times momentarily, only with dates, so it's not an issue.